public/style.css

```
@@ -1,51 +1,79 @@
-@import url('./style/house.css');
-@import url('./style/config.css');
-
-/* ============================================
-   RESPONSIVE STYLES (load last)
-   ============================================ */
-@import url('./responsive.css');
+:root {
+  /* defaults seguros */
+  --fx-bg: #0b0e14;
+  --fx-fg: #e6e6e6;
+  --fx-focus: currentColor;
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)